cisagov/ScubaGear
# Only import SCB markdowns needed for ScubaGear invocation

#  #

## 🗣 Description ##

Only import the SCB markdown needed to execute the current invocation of ScubaGear. Added additional error handling to Import of SCB.



## 💭 Motivation and context ##

Other product SCB info is not needed if not requested in ProductNames.  Added benifit is a error in a single SCB will not impact all products.  Additional error handling reports SCB parsing errors but continues processing.





Closes #520 

## 🧪 Testing ##

- Execute Unit test (all should pass)
- Execute ScubaGear with all products (see Errors for MS.AAD.7.3v1 unless aad.md corrected)
- Execute MarkdownCheck (i.e., Invoke-Pester -Path .\Testing\Functional\MarkdownChecks\ -Output Detailed).  ! test may fail if aad.md is not corrected. 







## ✅ Pre-approval checklist ##






      in code comments.
      to reflect the changes in this PR.
- [X] All new and existing tests pass unless expected as stated above. 

## ✅ Pre-merge checklist ##






## ✅ Post-merge checklist ##
